build/webpack/server.js

```diff
@@ -17,15 +17,15 @@ logger.error = function error() {
 };
 
 const options = {
-  publicPath: 'http://localhost:3000/app/',
+  publicPath: 'http://localhost:3001/app/',
```

Comment:
Can you explain the different ports 3001 vs 3000? I don't follow 🤔

Reply:
(question applies to all the different places these are changed throughout the PR)

Reply:
In the development mode, the webask development server and the proxy port (the actual hapi server port) were reversed. Due to this, webtask was unable to proxy requests to hapi, which was making the development mode broken. I reversed them.

Reply:
Note: This shouldn't affect production in any way.